### PR TITLE
App Structure

### PR DESCRIPTION
**Pull request by kevin-carlson**
Basic app structure created. Files added:

1. components/NavigationBar.js - basic navigation bar
2. pages/Blog.js - page used for viewing of blogs
3. pages/Create.js - create blog page with title and subtitle inputs
4. pages/Home.js - home page with basic information and login/get started button
5. pages/Profile.js - initialize file for user profiles

Edited App.js with basic page routing.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,59 @@
-import logo from "./logo.svg";
 import "./App.css";
 import { SkynetProvider } from "./state/SkynetContext";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
```
Comment (dghelm) [inline src/App.js:3]: You'll want to change "BrowserRouter" to "HashRouter" for Skynet Deployments. (At least until our support for this rolls out in the next couple of weeks.)

File: src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+          Skynet prioritizes privacy and security to empower users.
+        </Typography>
+      </Box>
+      <Box display={"flex"} justifyContent={"center"} style={{ margin: 20 }}>
+        <Button
+          size={"large"}
+          color={"primary"}
+          variant={"contained"}
+          style={{ width: "30%" }}
+          onClick={openSkynetHomePage}
```
Comment (dghelm) [inline src/pages/Home.js:78]: I think for a link you might just want to stick with the props ` target="_blank" href="https://siasky.net/"`

File: src/pages/Home.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
+import React, { useContext } from "react";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import Divider from "@material-ui/core/Divider";
+import Box from "@material-ui/core/Box";
+import { SkynetContext } from "../state/SkynetContext";
+import { Link } from "react-router-dom";
+
+export default function Home(props) {
```
Comment (MSevey) [inline src/pages/Home.js:10]: REQ: comment explaining purpose of the function

File: src/pages/Create.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import Container from "@material-ui/core/Container";
+import TextField from "@material-ui/core/TextField";
+import Grid from "@material-ui/core/Grid";
+
+export default function Create(props) {
```
Comment (MSevey) [inline src/pages/Create.js:6]: REQ: comment explaining purpose of the function

File: src/pages/Blog.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React, { useState, useEffect } from "react";
+import Container from "@material-ui/core/Container";
+
+export default function Blog(props) {
```
Comment (MSevey) [inline src/pages/Blog.js:4]: REQ: comment explaining purpose of the function

File: src/components/NavigationBar.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+import Add from "@material-ui/icons/Add";
+import { Link } from "react-router-dom";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+}));
+
+export default function NavigationBar(props) {
```
Comment (MSevey) [inline src/components/NavigationBar.js:17]: REQ: comment explaining purpose of the function

File: src/components/NavigationBar.js [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+import Typography from "@material-ui/core/Typography";
+import AppBar from "@material-ui/core/AppBar";
+import IconButton from "@material-ui/core/IconButton";
+import Toolbar from "@material-ui/core/Toolbar";
+import { makeStyles } from "@material-ui/core/styles";
+import AccountCircle from "@material-ui/icons/AccountCircle";
+import Add from "@material-ui/icons/Add";
+import { Link } from "react-router-dom";
+
+const useStyles = makeStyles((theme) => ({
```
Comment (MSevey) [inline src/components/NavigationBar.js:11]: REQ: comment explaining what/why you are defining here

File: src/App.js [hunk 8]
```diff
@@ -1,27 +1,59 @@
...
+import NavigationBar from "./components/NavigationBar";
+import React from "react";
+import Home from "./pages/Home";
+import Create from "./pages/Create";
+import Blog from "./pages/Blog";
+import { ThemeProvider, createTheme } from "@material-ui/core/styles";
+
+const theme = createTheme({
```
Comment (MSevey) [inline src/App.js:11]: REQ: comment explaining what this const is.

File: src/App.js [hunk 9]
```diff
@@ -1,27 +1,59 @@
...
+    primary: {
+      main: "#00C65E",
+    },
+    secondary: {
+      dark: "#0D0D0D",
+      main: "#EBEFEE",
+    },
+  },
+});
+const themeDark = createTheme({
```
Comment (MSevey) [inline src/App.js:22]: REQ: comment explaining what this const is.

**Review by MSevey — CHANGES_REQUESTED** [review]

**Pull request merged**